tools/example-showcase/src/main.rs

```diff
@@ -170,29 +175,30 @@ fn main() {
 
             let mut extra_parameters = vec![];
 
-            match (manual_stop, screenshot) {
-                (true, true) => {
+            match (stop_frame, screenshot_frame) {
+                (0, 0) => (),
+                (0, _) => {
                     let mut file = File::create("example_showcase_config.ron").unwrap();
                     file.write_all(
-                        b"(setup: (fixed_frame_time: Some(0.05)), events: [(100, Screenshot)])",
+                        format!("(setup: (fixed_frame_time: Some({fixed_frame_time})), events: [({screenshot_frame}, Screenshot)])").as_bytes(),
```

Comment:
How painful would it be to add `ron` as a dependency and serialize this instead of hardcoding the string? If [`CiTestingConfig`](https://github.com/bevyengine/bevy/blob/383314ef627da6654588a704959fc92c67770a52/crates/bevy_dev_tools/src/ci_testing.rs#L20) ever changes, I worry that it won't be caught immediately using this method.

Reply:
I would prefer not, that would mean maintaining those structs definition in multiple places...
once https://github.com/bevyengine/bevy/pull/13248 is ready, those should be part of CI anyway